tools/deprecate_version/deprecate_version.sh

```diff
@@ -1,15 +1,14 @@
 #!/bin/bash
 
+. tools/shell_utils.sh
+
 set -e
 
 SCRIPT_DIR=$(realpath "$(dirname "$0")")
-BUILD_DIR=build
-VENV_DIR=build/deprecate_version
+BUILD_DIR=$SCRIPT_DIR/build
```

Comment:
Why is this now moved out of `build`?

Reply:
I was trying to figure out where this would go and it appears it was at the top level. Is that correct? If so, `build` can't be created because it already exists as [`BUILD`](https://github.com/envoyproxy/envoy/blob/976fd328b0b107b1313c4ef668bfcf037940b83a/BUILD) for osx.

Reply:
Make it `build_tools` then.. I think this fix is orthogonal to the PR though, you can also leave it what it was before (yes, broken on OS X).

Reply:
Sounds good. Updated the dir to `build_tools` :)